### ISSUE TITLE
Log console to Discord channel

### ISSUE THREAD

**Issue by Metacinnabar** [xNarnia/TCR-TerrariaChatRelay#12]
## Overview
Log all messages from the server console to (a) specified Discord channel(s) through the same chat bridge bot.

**Comment by xNarnia** [comment]
Interesting, I'll think about how to make this make sense

**Comment by Metacinnabar** [comment]
Surely just inject into Console.WriteLine

**Comment by xNarnia** [comment]
Yeah for sure! The way the system works right now is that it takes single channel id's with no filtering for what goes where. So, I think a rework for the channel id system will probably have to be done to make it make the most sense. I'll try to get this in soon, thanks for your patience.

**Comment by xNarnia** [comment]
I think for now I'll just hold another array that you can set to receive Console output to get you this functionality

**Comment by xNarnia** [comment]
Added in TerrariaChatRelay v2.6
